etp-backend/src/main/sql/solita/etp/db/whoami.sql

```diff
@@ -6,9 +6,7 @@ SELECT k.id id, k.etunimi etunimi, k.sukunimi sukunimi, k.email email,
 FROM kayttaja k
 LEFT JOIN laatija l ON l.id = k.id
 WHERE (not k.passivoitu) AND
-      ((k.email IS NOT NULL AND k.email  = :email) OR
-      (k.henkilotunnus IS NOT NULL AND k.henkilotunnus = :henkilotunnus) OR
-      (k.cognito_id IS NOT NULL AND k.cognito_id = :cognitoid) OR
+      ((k.henkilotunnus IS NOT NULL AND k.henkilotunnus = :henkilotunnus) OR
       ((k.virtu$localid IS NOT NULL AND k.virtu$localid = :virtu_localid) AND
        (k.virtu$organisaatio IS NOT NULL AND k.virtu$organisaatio = :virtu_organisaatio)))
```

Comment:
tämän voisi samalla muuttaa selkeämmäksi eli ottaa pois noi turhat 'IS NOT NULL' lausekkeet